modules/siddhi-launcher/src/main/java/org/wso2/siddhi/sdk/launcher/run/SiddhiRun.java

```diff
@@ -37,24 +37,21 @@ public SiddhiRun() {
 
     public void runSiddhi(String siddhiApp, String inputFile) throws InterruptedException {
 
-        while (true) {
-            try {
-                //Generating runtime
-                SiddhiAppRuntime siddhiAppRuntime = siddhiManager.createSiddhiAppRuntime(siddhiApp);
-
-                //Starting event processing
-                siddhiAppRuntime.start();
-
-                if (!(inputFile == null || inputFile.equalsIgnoreCase(""))) {
-                    InputFeeder inputFeeder = new InputFeeder(siddhiAppRuntime, inputFile);
-                    //starting input feeder
-                    inputFeeder.start();
-                    break;
-                }
-            } catch (Exception e) {
-                log.error("Internal Siddhi Error Occurred: " + e);
-                break;
+        try {
+            //Generating runtime
+            SiddhiAppRuntime siddhiAppRuntime = siddhiManager.createSiddhiAppRuntime(siddhiApp);
+
+            //Starting event processing
+            siddhiAppRuntime.start();
+
+            if (!(inputFile == null || inputFile.equalsIgnoreCase(""))) {
```

Comment:
Should rather use StringUtils.isEmpty if the dependency is available. It's more readable.

Reply:
Usually, all the siddhi related repositories do not have org.apache.commons.lang3 dependency. Hence followed this way.